_posts/2020/11/2020-11-23-python-repository.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+date: 2020-11-23
+title: "Some name"
+author: olenagerasimova
+description: This blog post shows how binary and pypi repositories can be used.
+keywords:
+  - artipie
+  - private
+  - pypi
+  - python
+  - repository
+  - registry
+  - binary
+---
+
+Being a student I dreamed about something like Artipie, I needed some private centralized storage 
+where I could have kept both binary or text data files and Python packages. We were conducting a research
+and used to work with a lot of data from experimental measurements and used Python to process and 
+visualise them. My university colleagues from that time were mathematicians, they do not have strong
+development background and were absolutely fine with passing data and code via flash drive or email.
+All my efforts to infiltrate VCS (GitHub, for example, provides free Pro accounts for students 
```

Comment:
@olenagerasimova "infiltrate" is not right word here, it means to get inside some organization or facility as a spy. I think in such cases it's more common to say about adoption of something by organization, ex "efforts to adopt VCS in university" or "efforts to convince colleagues to adopt VCS"

Reply:
@olegmoz thanks, I think [inculcate](https://www.lingvolive.com/ru-ru/translate/en-ru/inculcate) will be correct here